### DISCUSSION TITLE
Use `next/image`

### DISCUSSION THREAD

**Discussion by styfle** [heroui-inc/heroui#165] [Help] [answered]
It would be great if NextUI used the built-in [`next/image`](https://nextjs.org/docs/api-reference/next/image) component, that way it could take advantage of [Image Optimization](https://nextjs.org/docs/basic-features/image-optimization).

**Comment by elitan** [comment]
I'm not sure it's a good idea because of this: https://github.com/vercel/next.js/discussions/19065

**Comment by jrgarciadev** [comment, marked as answer]
Hey @styfle all NextUI components has a polymorphic property called `as` so you can pass the Next.js Image component and the NextUI `Image` component will support and behave as the Next.js `Image` component here's an example -> https://stackblitz.com/edit/nextjs-yie9lo?file=pages/index.js
